cypress-tests/cypress/e2e/spec/Payout/00005-SavePayout.cy.js

```diff
@@ -229,10 +229,10 @@ describe("[Payout] Saved Bank transfer", () => {
         cy.listCustomerPMCallTest(globalState);
       });
 
-      it("[Payout] [Bank transfer] [SEPA] Fulfill using Token", () => {
+      it("[Payout] [Bank transfer] [sepa_bank_transfer] Fulfill using Token", () => {
```

Comment:
nit: can this be capitalised?

```suggestion
      it("[Payout] [Bank transfer] [SEPA_BANK_TRANSFER] Fulfill using Token", () => {
```

even on line `179`.